### DISCUSSION TITLE
Building Progression

### DISCUSSION THREAD

**Discussion by stevew00dy** [sunflower-land/sunflower-land#1619] [Ideas]
Currently in Sunflower Isles the concept around buildings is to lock game features behind a level gate. 

A progression loop around this concept is to upgrade buildings over time. For example a fire pit can evoke into an oven into a kitchen. 

With each progression it will unlock new abilities within the building.

My proposal is with regards to volume and rarity. 

I propose that when a building is placed it is a common building which grants a limited number of tasks, item space and also timers. By adding additional resources (burn) players can evolve their building (depending on their bumpkin level) to improve the overall stats. 

The progression system would be common, uncommon, rare, epic, legendary. 

Once legendary the building would acquire an animation aura that would make it stand out from others. 

This would provide an additional progression system, an element of value awarded from an obvious grind.

**Comment by FarmerKuenster** [comment]
This is a really cool idea, It lets players have another choice of what to do next in the game. Maybe even once you upgrade a common building to uncommon you can then build another common version of that building if you want. It stinks having to wait to only do one cake at a time..

**Comment by cryptosmikey** [comment]
Great idea, I like how people will have to progress through rarities instead of being able to buy and build. Great idea!
